### DISCUSSION TITLE
pecha kucha with pympress

### DISCUSSION THREAD

**Discussion by yannis1962** [Cimbali/pympress#230] [Q&A] [answered]
Hi,
is it possible to use pympress for pecha kucha, i.e., pympress displaying automatically the next slide every 20 seconds?

If yes, please let me now how.

If not, with all the features pympress has already this shouldn't be difficult to implement, maybe adding a few lines of code somewhere? (I need it for tomorrow…)

Thanks in advance,

Yannis

**Comment by Cimbali** [comment, marked as answer]
I’m not sure what pecha kucha is − but if you want automatic slide transitions you can try installing ~the latest beta~ v1.7+, it’s under Navigation > Automatic navigation. You can use `p` to pause the process if you want.
